### DISCUSSION TITLE
Delete Filament Resource

### DISCUSSION THREAD

**Discussion by ousid** [filamentphp/filament#3508] [Ideas]
I'm just heading to make a PR for the ability to delete/remove, something like

```bash
php artisan make:filament-destroy NameOfTheResource
```

By running the `destroy` command, it will delete the resource and the pages a long with it.

Is it a feature worth creating, or just a waste of time?

Your feedback is much appreciated.

cc, @danharrin

Thanks.

**Comment by SamHep0803** [comment]
Honestly IMO this is a useless feature as it's literally just as simple as deleting the resource file and folder, I think a command to do this is just a little unnecessary.

**Reply by ousid** [reply]
I thought following `livewire convention` (If this term fit) as a UX would be a great addition, but who knows...

**Comment by haztakki** [comment]
Yeah, I kind of agree with @SamHep0803 here, I would never use that. It's easier to just delete the file / resource folder using Cmd + Click.

**Reply by ousid** [reply]
Maybe you're both right, I'm a terminal guy, I felt optimistic about this one, but if 90% of the users don't use it, yea it's a useless addition.

**Reply by SamHep0803** [reply]
Look that's just my opinion though, no one's stopping you from submitting the PR, and if it gets accepted then it gets accepted :)

**Reply by ousid** [reply]
Thanks, @SamHep0803, for your feedback,

I'm just making sure before I waste an hour or so on a PR I know It will get reject :)

**Comment by danharrin** [comment]
Yeah I agree with the other relies, its easier to just delete the files, as they're all in one place.

I think Livewire has the command as the component class and view are in distant directories and need to be removed from each place.

**Reply by ousid** [reply]
Yea, totally agree about livewire use case.

Thanks for letting me know, @danharrin.
